.gitignore

```diff
@@ -0,0 +1,52 @@
+# CMake and build files
+*.o
+build/
+CMakeFiles/
+Makefile
+*.cmake
+CMakeCache.txt
+cmake_uninstall.cmake.in
+
+# Headers created when running cmake
+inc/defines.h
+*_header.h
+*_init.h
+
+# Libraries and pilight executables
+*.a
+*.so
+pilight-control
+pilight-daemon
+pilight-debug
+pilight-flash
+pilight-raw
+pilight-receive
+pilight-send
+pilight-sha256
+pilight-uuid
+*.exe
+
+# JSON files in base directory - often used for local testing of different configs
+/*.json
+
+# IDE project files
+.project
+.settings/
+.cproject
+.idea/
+*.iml
+*.ipr
+*.iws
+
+# General os files
+*~
+*.bak
+*.swp
+.directory
+.DS_Store
+.DS_Store?
+._*
+.Spotlight-V100
```

Comment:
Why?

Reply:
IIRC, it's some index file that OSX uses

Reply:
Wierd stuff. And if so, why isn't there a pilight OSX build?

Reply:
Dunno, nobody has tested it yet I suppose. I guess I could remove it for now until people start compiling pilight on mac's 😆

Reply:
Yes

Reply:
Done